src/NHapi.Base/Model/Varies.cs

```diff
@@ -34,29 +34,39 @@
 
 namespace NHapi.Base.Model
 {
-   /// <summary> <p>Varies is a Type used as a placeholder for another Type in cases where 
+   /// <summary>
+   /// Varies is a Type used as a placeholder for another Type in cases where 
    /// the appropriate Type is not known until run-time (e.g. OBX-5).  
    /// Parsers and validators may have logic that enforces restrictions on the 
-   /// Type based on other features of a segment.</p>  
-   /// <p>If you want to set both the type and the values of a Varies object, you should
+   /// Type based on other features of a segment.
+   /// <para>
+   /// If you want to set both the type and the values of a Varies object, you should
    /// set the type first by calling setData(Type t), keeping a reference to your Type, 
-   /// and then set values by calling methods on the Type.  Here is an example:</p>
-   /// <p><code>CN cn = new CN();<br>
-   /// variesObject.setData(cn);<br>
-   /// cn.getIDNumber().setValue("foo");</code></p>
+   /// and then set values by calling methods on the Type.
+   /// </para>
    /// </summary>
-   /// <author>  Bryan Tripp (bryan_tripp@users.sourceforge.net) 
-   /// </author>
+   /// <example>
+   /// <code>
+   /// CN cn = new CN();
+   /// variesObject.setData(cn);
+   /// cn.getIDNumber().setValue("foo");

```

Comment:
`cn.IDNumber.Value = "foo";`

again not part of the fix build warnings, I think i will create a seperate ticket to improve the xml comments ahead of documentation generation.